packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -158,3 +158,16 @@ export const PlatformUrls: TPlatformUrls = {
         live: 'https://dx.deriv.com',
     },
 };
+
+export const CFD_PLATFORMS = {
```

Comment:
We should standardize one, camelcase or snake case. So far in tradershub we using camelcase for all

Reply:
I think this is okay, since they are enums

Reply:
We already have these constants in line 48 of the same file
<img width="269" alt="image" src="https://github.com/binary-com/deriv-app/assets/82078941/8165d899-e310-46b0-9599-bc6bb76c18ae">

Reply:
I am using these now
Removed the added ones